docs/CNAME

```diff
@@ -1 +0,0 @@
-buildpacks.io
```

Comment:
Basically, there are top-level directories called `docs` and `resources`. I'm unsure if they're actually needed or just leftover. I simply copied the real `docs` and `resources` dirs that are located in `src` and replaced the outer ones with those too. I'm not sure this is the right thing to do, so some guidance would be helpful.

This deletion is a result of this file not existing in the original dir I copied from...